server/src/main/java/org/elasticsearch/cluster/ClusterState.java

```diff
@@ -283,12 +279,12 @@ public DiscoveryNodes nodesIfRecovered() {
         return blocks.hasGlobalBlock(STATE_NOT_RECOVERED_BLOCK) ? DiscoveryNodes.EMPTY_NODES : nodes;
     }
 
-    public Map<String, TransportVersion> transportVersions() {
-        return this.transportVersions;
+    public Map<String, VersionsWrapper> versionsWrappers() {
+        return this.versionsWrappers;
     }
 
-    public TransportVersion getMinTransportVersion() {
```

Comment:
I suspect it'd help to keep `getMinTransportVersion()` and add new `getMinXxxVersion()` accessors as we expand `VersionsWrapper`. I can't really think of a good situation for exposing the whole `VersionsWrapper`.